prefetch.bs

```diff
@@ -381,8 +380,6 @@ These algorithms are based on [=process a navigate fetch=].
 
         :  [=request/URL=]
         :: |prefetchRecord|'s [=prefetch record/URL=]
-        :  [=request/referrer policy=]
-        :: |prefetchRecord|'s [=prefetch record/referrer policy=]
```

Comment:
Why remove this? I would expect that we will still want it to be able to have a value other than the document's default in the future, such as the mechanism in #167 and inferring from `<a referrerpolicy>`.

Reply:
Indeed, #167 might add something like this back. However, what we add back might not be the same, e.g. it'd probably be more like an optional referrer policy so that we get the correct fallback behavior if it's absent. So my plan was to remove for now to align with the implementation plans, and then evaluate the solution for #167, or for document rules, independently.

That said, I see there's a PR for document rules now, and I hope to work on #167 soon, so if we want to leave this un-merged and see how those shape up before merging, I'd be OK with that.

Reply:
It feels like it'd be pretty close to what's here now, except possibly making it optional (depending on whether we want the document's referrer policy at the point where the speculation is initiated or where the speculation is detected -- the two will seldom if ever differ). If you felt strongly about landing this as-is I think it'd be okay, but it just feels odd to remove and then re-add something very similar.

Reply:
I'm getting pretty nervous about how far the explainers and specs are from what we've currently implemented, so I'd like to merge this and then rebuild it as appropriate :).